spring-cloud-gcp-data-spanner/src/test/resources/logback-test.xml

```diff
@@ -1,4 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <configuration>
+	<appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+		<!-- encoders are assigned the type
+		ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
+		<encoder>
+			<pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+		</encoder>
+	</appender>
 
-  <root level="warn"/>
```

Comment:
I think `root` still has to be there:
```
<root level="warn">
	<appender-ref ref="STDOUT"/>
</root>
```

No? Does it work properly even without that?
I mean do we meet the general purpose of this PR if we don't have `root`?

Thanks